static/scripts/dashboard/dashboardCtrl.js

```diff
@@ -1418,7 +1419,26 @@
       );
     }
 
+    function getMsgSystemIntegration(update, next) {
```

Comment:
Why can't you use `isInitialized` from `systemSettings`?  This doesn't accurately determine if it's initialized; it may have failed.